animareflection/ui
# Update `Card` Component

## Description

##### Task link: https://trello.com/c/kwhaYjTC/227-card-patches

Updated `Card` component to remove need for `use client` directive in downstream NextJS apps as well as updated the component to allow for a wider range of children beyond just `CardHeader`, etc.

## Test Steps

1) Verify structure of component is sound.
2) Verify that downstream NextJS apps are not required to use `use client` directive due to `Card` component anymore (example app is a great spot to check, removed the directive from `CardDemo`).
